linux_sql/scripts/host_usage.sh

```diff
@@ -0,0 +1,29 @@
+psql_host=$1
+psql_port=$2
+db_name=$3
+psql_user=$4
+psql_password=$5
+
+if [ $# -ne  5 ]; then
+  echo "Bad Arguments"
```

Comment:
Same as above. It's not that the arguments are bad, it's that there is the wrong number of them, yes?